Gopkg.lock

```diff
@@ -331,7 +331,7 @@
   branch = "master"
   name = "github.com/google/go-github"
   packages = ["github"]
-  revision = "218ecc238cf77bbaa2e16a54cada252ec87e7d6e"
+  revision = "c988f775700b9ab14b5acb7502046fa341daf82d"
 
```

Comment:
@costinm why is the bot updating gopkg.lock ? That too, without supervision ? If someone pinned a sha in gopkg.toml, what happens then?